docs/docs/configuration.mdx

```diff
@@ -2,31 +2,81 @@
+| --vault-token-path | APPROZIUM_VAULT_TOKEN_PATH |            | Optional, if set it will cause the latest Vault token to always be pulled from the given file. This option takes precedence over --vaulttoken                                                                                                                                                                                                                             |
+| --vault-token      | APPROZIUM_VAULT_TOKEN      |            | Optional, if set it will be used at the Vault token.                                                                                                                                                                                                                                                                                                                      |
+| --vault-addr       | APPROZIUM_VAULT_ADDR       |            | Required if using Vault as the secrets manaager.                                                                                                                                                                                                                                                                                                                          |
+| --aws-region       | APPROZIUM_AWS_REGION       |            | Required if using AWS Secrets Manager as the secrets manager.                                                                                                                                                                                                                                                                                                             |
+
+## Misc Flags
+
+| Name, shorthand    | Environment variable       | Default    | Description                                                                                                                                                                                                                                                                                                                                                               |
+|--------------------|----------------------------|------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --config           | APPROZIUM_CONFIG_FILE_PATH | false      | Optional, set it to path containing a YAML config file called `approzium.config.yml`                                                                                                                                                                                                                             |
```

Comment:
config file no longer has to be named `approzium.config.yml` with #180, right?

Reply:
Good call! https://github.com/cyralinc/approzium/pull/181/commits/9e00ab04db20c010345d50a4df1402754c8a662f